rakelib/arp_seed_data.rake

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require_relative '../lib/accredited_representation/seed_data'
+
+namespace :arp do
+  desc 'Seed VerifiedRepresentative data for staging'
+  task :seed_representative, [:test_rep_email] => [:environment] do |_, args|
+    raise 'No representative email provided' unless args[:test_rep_email]
+
+    # Create VerifiedRepresentative and AccreditedIndividual for logging into accredited_representative_portal
+    ogc_registration_number = '123'
```

Comment:
I would recommend that you don't hard code these, but include these as args in the rake task itself